src/components/component/env-variables.js

```diff
@@ -39,20 +41,21 @@ const EnvVariables = ({ component, includeRadixVars }) => {
 
   return (
     <React.Fragment>
-      <Typography variant="h4">Environment variables</Typography>
-      {hasRadixVars && (
-        <Typography variant="body_short">
-          (* automatically added by Radix)
-        </Typography>
-      )}
-      {envVarNames.length === 0 ||
-        (!includeRadixVars && envVarNames.length - radixVarNames === 0 && (
+      <div className="env-var--header">
+        <Typography variant="h4">Environment variables</Typography>{' '}
+        {hasRadixVars && (
           <Typography variant="body_short">
-            This {component.type === 'job' ? 'job' : 'component'} uses no
-            environment variables.
+            ( <Logo height="24px" width="24px" /> automatically added by Radix )
           </Typography>
-        ))}
-      {envVarNames.length > 0 && (
+        )}
+      </div>
+      {envVarNames.length === 0 ||
+      (!includeRadixVars && envVarNames.length - radixVarNames === 0) ? (
+        <Typography variant="body_short">
+          This {component.type === 'job' ? 'job' : 'component'} uses no
```

Comment:
component type is defined in /models/component-type/index.js
Perhaps better to import the componentType const from this file and use `component.type === componentType.job` instead?